src/utils/preprocessing.py

```diff
@@ -394,3 +404,15 @@ def pcd_processing_gapnet(pcd_paths):
     pointclouds = pointclouds.reshape((-1, 1024, 3))
 
     return pointclouds
+
+
+def posenet_processing(filename):
+    image = cv2.imread(filename)
+    if scan_type['Standing_front'] in filename or scan_type['Standing_back'] in filename or scan_type['Standing_360'] in filename:
```

Comment:
This could be refactored the same way we do it here: https://github.com/Welthungerhilfe/cgm-ml/blob/master/src/common/result_generation/blur.py#L83